Gui/FileTypeMainWindow_win.cpp

```diff
@@ -87,26 +87,36 @@ DocumentWindow::~DocumentWindow()
 // —— public slots ——————————————————————————
 // —— protected slots —————————————————————————
 // —— events ————————————————————————————
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
 bool
-DocumentWindow::winEvent(MSG *message,
-                         long *result)
+DocumentWindow::nativeEvent(const QByteArray& eventType, void* message, long* result)
 {
```

Comment:
avoid double curly braces, which may confuse indenteds. I would prefer having two separate function definitions (will make the change)